src/components/Highlights.vue

```diff
@@ -1,26 +1,181 @@
 <template>
   <div class="highlights-bar">
-    <div class="betas">JSE Index:{{ jseIndex }}</div>
-    <div class="betas">{{ industryIndices[1] }}: {{ industryBetas[1] }}</div>
-    <div class="betas">{{ industryIndices[2] }}: {{ industryBetas[2] }}</div>
+    <div class="betas">
+      All Share Index (30 Days): {{ index1 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
```

Comment:
There is a nice short syntax for v-bind where you can simply do : . I would implement that as it's the standard for VUE and makes it look a lot neater

e.g. v-bind:class = :class

Reply:
I'll look it up, thanks!